addons/sourcemod/scripting/vsh/abilities/ability_teleport_swap.sp

```diff
@@ -132,7 +146,16 @@ methodmap CTeleportSwap < SaxtonHaleBase
 			
 			float vecAng[3];
 			GetClientEyeAngles(this.iClient, vecAng);
-			if ((vecAng[0] < -60.0) && (this.iCharge >= this.iMaxCharge))
+			
+			//Cap value to prevent impossible angle
+			float flAngleRequirement = this.flEyeAngleRequirement;
+			if (flAngleRequirement > 90.0)
+				flAngleRequirement = 90.0;
+				
+			else if (flAngleRequirement < -90.0)
+				flAngleRequirement = -90.0;
+			
+			if ((vecAng[0] <= flAngleRequirement) && (this.iCharge > 1))
```

Comment:
I see copy-paste from brave-jump, `this.iMaxCharge` replaced to `1`

Reply:
u right